### ISSUE TITLE
New strings on Source always need editing

### ISSUE THREAD

**Issue by Cussa** [WeblateOrg/weblate#1783]
Here on my company we are starting to use the Weblate. We have a process where the developer create the new strings on the English file. After that, a English reviewer will take a look on the new strings to see if there is some kind of gramatical or other problem. After that, the translators should go to weblate and translate them.

My questions are:
1. Is there any way to set that new strings (on source file) always need editing?
2. Is there any way to notify the translators only after the source strings are already edited?

**Comment by nijel** [comment]
1. Yes, just add add-source-review flag to the component flags (see https://docs.weblate.org/en/latest/admin/checks.html?highlight=mark-source-review#fine-tuning-existing-checks).
2. This is currently not available.

Setting the flag is a bit obscure and this should be probably turned to addon once we support them (see https://github.com/WeblateOrg/weblate/issues/1353).

**Comment by lagartoflojo** [comment]
Hi @nijel, I'm a bit confused about the usage of the `add-source-review` flag. If its intention is to make it easy for a proofreader to figure out which strings are new, I expected there to be a link to "source strings pending review" in the source strings page (ie. https://mywebla.te/projects/xxx/yyy/source/). Am I missing something?

**Comment by nijel** [comment]
Hmm, this is probably a bit confusing, but this happens on the real source language (eg. https://hosted.weblate.org/projects/androbd/strings/en/). The source page (eg. https://hosted.weblate.org/projects/androbd/strings/source/) is more overview of things which might indicate problems in source language. 

This whole confusion comes from support for both monolingual and bilingual translations - the bilingual ones do not have real source translation and those checks had to be placed somewhere. Later editing of source strings for monolingual was add (and is optional), but this was never fully integrated...

**Comment by lagartoflojo** [comment]
Thanks for the explanation. We are just starting to use Weblate, so I apologize if I'm thick-headed about some things 😅 
In our case, we use PO/POT (so a bilingual format), and we have a PO file for every language, including the original (English). Our English PO only has the `msgid`'s, the `msgstr`'s are empty (`""`).
If I understand you correctly, with the `add-source-review` flag, when the English PO changes, the new/changed strings will be marked as "waiting for review", even if they are not marked as `fuzzy` in the actual PO file? Then I would be able to go to the project homepage, and click on the percentage number under the "Review" column?

**Comment by nijel** [comment]
The `add-source-review` flag will not work for bilingual formats, sorry. At least not until we have something like https://github.com/WeblateOrg/weblate/issues/1822.

**Comment by lagartoflojo** [comment]
@nijel Understood, thank you!

**Comment by nijel** [comment]
I've just implemented the addon for this, it will replace the flag in Weblate 2.19.

Thank you for your report, the issue you have reported has just been fixed. 

* In case you see problem with the fix, please comment on this issue. 
* In case you see similar problem, please open separate issue.
* If you are happy with the outcome, consider supporting Weblate by [donating](https://weblate.org/donate/).
